main.js

```diff
@@ -10,53 +10,53 @@ const FEATURES = [
   },
   {
     title: '`async` as a keyword in 2018 edition',
-    stablized: {
+    stabilized: {
       version: '1.28',
-      pr: '50307',
-    },
+      pr: '50307'
```

Comment:
I tend to keep the trailing comma for multi-line expressions when it's not JSON.